### DISCUSSION TITLE
Options for Formatted Value Display

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#9040] [Feature Requests]
Options for...
* Enable Links — (eg: url or email) when enabled, shows a `launch` icon (18px) on hover, clicking launches link
    * External links should auto open in new tab
* Enable Copy Button — when enabled, shows a `copy` icon (18px) on hover, clicking copies the value
* Better number formatting for `9,000`, not `9000` (could use locale for formatting)
    * Maybe we can pull this from the user locale?
* Prepend Text (useful for `$123,456`)
* Appended Text (useful for `123,456 Widgets`)
* Icon After — maybe conditionally disables when using link/copy?
* Conditional Formatting Repeater
    * Operator Dropdown
    * Value Input
    * Color Chooser
    * Icon Chooser

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
